Description: Enable +derive for serde to run tests
Last-Update: 2025-09-09
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: ipnetwork/Cargo.toml
===================================================================
--- ipnetwork.orig/Cargo.toml
+++ ipnetwork/Cargo.toml
@@ -54,4 +54,5 @@
 [dependencies.serde]
 version = "1.0.200"
+features = ["derive"]
 optional = true
 
